### DISCUSSION TITLE
NetBeans 16 and existing preinstalled JDK

### DISCUSSION THREAD

**Discussion by jmccay-work** [apache/netbeans#5394] [Q&A] [answered]
I currently have NetBeans 16 installed on a WSL2 Linux Mint install.  I installed OpenJDK 11 on the Linux Mint.  I looked around the interface, but I can't find a way to add an existing instance of Red Hat OpenJDK 8 32 bit (exact version below) that is installed on the Windows side.  I have the C drive mounted, and I can open projects on the mounted drive.  Is there a way to add that version of Java to build projects?  I also have versions of Maven, Ant, Groovy, and Gradle installed on that drive.

- openjdk version "1.8.0_292"
- OpenJDK Runtime Environment (build 1.8.0_292-b10)
- OpenJDK Server VM (build 25.292-b10, mixed mode)

**Comment by mbien** [comment, marked as answer]
`tools -> java platforms` manages java installs.

but running a windows jdk on linux will probably not work that easily.

**Reply by jmccay-work** [reply]
It's already installed on the Windows machine.  The Linux is actually a Mint install installed on Windows through WSL2.  It will not let me select the directory the Java version I already have installed.  I should mention the already installed instance is Java 8 32 bit (Red Hat Open JDK for Windows).

**Reply by lkishalmi** [reply]
That's just bad. Either use NetBeans on Windows with Windows JDKs or on Linux with Linux JDK-s.
I really need to debug something on an other platform it could be a little but inconvenient, but usually a with a few exrta steps you can do remote execution/debugging between two platforms.

**Comment by jmccay-work** [comment]
I realized I don't need to compile the new project with Java 8 (32-bit) because the program will not interact with my company's software directly.  Thanks for the help.
